### ISSUE TITLE
[Bug] `checkhealth telescope._extensions.lazy_plugins` returns an error

### ISSUE THREAD

**Issue by pidgeon777** [polirritmico/telescope-lazy-plugins.nvim#3]
I see that there is a `checkhealth` command available:

![image](https://github.com/polirritmico/telescope-lazy-plugins.nvim/assets/13739892/c237cf5d-0a65-412f-89bd-21a6f8c4c1fd)

When I run it, the following error is reported:

```log
telescope._extensions.lazy_plugins: require("telescope._extensions.lazy_plugins.health").check()

- ERROR Failed to run healthcheck for "telescope._extensions.lazy_plugins" plugin. Exception:
  [string "require("telescope._extensions.lazy_plugins.h..."]:1: attempt to index a function value
```

**Comment by pidgeon777** [comment]
`checkhealth telescope` is ok:

```log
==============================================================================
telescope: require("telescope.health").check()

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 14.1.0 (rev e50df40a19)
- OK fd: found fd 9.0.0

===== Installed extensions ===== ~

Telescope Extension: `aerial` ~
- No healthcheck provided

Telescope Extension: `everything` ~
- No healthcheck provided

Telescope Extension: `fzf` ~
- OK lib working as expected
- OK file_sorter correctly configured
- OK generic_sorter correctly configured

Telescope Extension: `lazy_plugins` ~
- OK Lazy configuration file was found.
- OK Path to Lazy plugins spec table file found.

Telescope Extension: `projects` ~
- No healthcheck provided
```

**Comment by polirritmico** [comment]
Hi! Thank you very much for the bug report.
Yes, indeed, that check function shouldn't be exposed. As stated in the README, its usage is through `:checkhealth telescope`.

**Comment by polirritmico** [comment]
Closed by c9d251d18025468b6649e8489d3b6f3510027f03
